modules/javafx.web/src/test/java/test/javafx/scene/web/LocalStorageTest.java

```diff
@@ -40,19 +41,16 @@
 
 public class LocalStorageTest extends TestBase {
 
-    private static final File LOCAL_STORAGE_DIR = new File("LocalStorageDir");
+    private static final File LOCAL_STORAGE_DIR_PATH = new File("build/");
+    private static final File LOCAL_STORAGE_DIR = new File("build/localstorage");
 
     private static void deleteRecursively(File file) throws IOException {
         if (file.isDirectory()) {
             for (File f : file.listFiles()) {
                 deleteRecursively(f);
+                f.delete();
```

Comment:
This will miss deleting the original directory passed into this method. What was wrong with the old way of doing it?